tutorials/dataframe/df006_ranges.C

```diff
@@ -50,12 +50,12 @@ int df006_ranges()
    auto d_15_end = d.Range(15, 0);
    auto c_15_end = d_15_end.Count();
 
-   // We can use a stride too, in this case we pick an event every 3
+   // We can use a stride too, in this case we pick an event every 3 entries
    auto d_15_end_3 = d.Range(15, 0, 3);
    auto c_15_end_3 = d_15_end_3.Count();
 
    // The Range is a 1st class citizen in the RDataFrame graph:
```

Comment:
I am not sure people will understand `"1st class citizen"`. I would rather say:
```suggestion
// The Range here acts first on the (whole) RDataFrame graph.
```